exercises.ipynb

```diff
@@ -172,22 +1861,699 @@
     "\n",
     "HINT: Write a separate function to get the current exchange rates! That can be reused!"
    ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 26,
+   "metadata": {
+    "scrolled": true
+   },
+   "outputs": [],
+   "source": [
+    "import pandas as pd\n",
+    "assets = pd.read_csv('assets.csv')"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 28,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def current_ex_rate(url):\n",
+    "    request_exrate_latest = requests.get(url)\n",
+    "    rates_json = request_exrate_latest.json()\n",
+    "    return rates_json\n"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 53,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "def make_rates_df():\n",
```

Comment:
Functions should take arguments, not rely on global variables (rates_json)!! This doesn't even look like it will work.